### DISCUSSION TITLE
Ubuntu / Debian

### DISCUSSION THREAD

**Discussion by massimopeca** [rustdesk/rustdesk#9388] [General]
Hello everyone,
Is Rustdesk fully compatible with Debian?
Among the available versions, I see that only Ubuntu is mentioned, yet it is Ubuntu that is derived from Debian and not the other way around. And it is also less widespread than Debian.
Why is Ubuntu mentioned instead of Debian?
Thanks
Hi.

**Comment by delinuxco** [comment]
You can use the Ubuntu one with Debian just fine.
